x/profiles/types/events.go

```diff
@@ -1,11 +1,18 @@
 package types
 
 const (
-	EventTypeProfileSaved   = "profile_saved"
-	EventTypeProfileDeleted = "profile_deleted"
+	EventTypeProfileSaved            = "profile_saved"
+	EventTypeProfileDeleted          = "profile_deleted"
+	EventTypeDTagTransferRequest     = "dtag_transfer_request"
+	EventTypeDTagTransferReqAccepted = "accepted_transfer_request"
```

Comment:
I think it would be more appropriate to rename this event to 

```suggestion
	EventTypeDTagTransferAccept = "dtag_transfer_accept"
```

To make it more coherent wit the request one